src/pages/AddJob.jsx

```diff
@@ -42,6 +43,73 @@ export default function AddJob() {
     }
   );
 
+  useEffect(() => {
+    let hasdraft = window.localStorage.getItem("draft");
+    if (hasdraft !== null && toastid.length === 0) {
```

Comment:
i believe it should be `undefined`...

Reply:
I tried in console log and it should be null...